support-frontend/assets/pages/[countryGroupId]/components/thankYouComponent.tsx

```diff
@@ -113,6 +113,11 @@ export function ThankYouComponent({
 		case 'DomesticMonthly':
 		case 'RestOfWorldMonthlyV2':
 		case 'DomesticMonthlyV2':
+		case 'Everyday':
+		case 'Sixday':
+		case 'Weekend':
+		case 'Saturday':
+		case 'Sunday':
```

Comment:
I'm going to refactor this code in a separate PR as contributionType doesn't really apply to paper or tier three subs